test/test_EKF_gps_yaw.cpp

```diff
@@ -88,26 +90,61 @@ TEST_F(EkfGpsHeadingTest, fusionStartWithReset)
 	_sensor_simulator.runSeconds(11);
 
 	// THEN: after a while the fusion should be stopped
-	// TODO: THIS IS NOT HAPPENING
 	EXPECT_FALSE(_ekf_wrapper.isIntendingGpsHeadingFusion());
 }
 
-TEST_F(EkfGpsHeadingTest, fusionStartWithoutReset)
+TEST_F(EkfGpsHeadingTest, yawConvergence)
```

Comment:
Not totally, the `fusionStartWithReset` function tests if the reset counter increases properly and that the fusion stops when GPS data stops for 10s. We could merge them together, but it would maybe test too many different things.